scripts/verify-permissions.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2021 Paul Keith <javelinanddart@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 and
+# only version 2 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# Modified by Ilya Danilkin <nezorflame@gmail.com> @ OpenGApps
+#
+# This script helps to fix the private permission issues caused by updated APKs.
+# See https://gitlab.com/MindTheGapps/vendor_gapps/-/blob/sigma/cicd/verify-permissions.py
+# for the original file.
+
+import errno
+from glob import glob
+import os
+import subprocess
+import sys
+from xml.etree import ElementTree
```

Comment:
Also do a check if these imported libraries are available in our checktools (or other graceful dealing with dependencies)